lib/autobuild/import/git.rb

```diff
@@ -293,6 +293,52 @@ def validate_importdir(package)
             git_dir(package, true)
         end
 
+        
+        # Return default local branch if exists, if not return nil
+        #
+        # @param [Package] package
+        def default_local_branch(package)
+            ls_local_string = run_git(package, 'symbolic-ref', "refs/remotes/#{@remote_name}/HEAD").first.strip
+            default_local_branch = ls_local_string.match("refs/remotes/#{@remote_name}/(.*)")
+            default_local_branch.nil? ? nil : default_local_branch[1]
+            rescue Autobuild::SubcommandFailed
```

Comment:
`rescue` should have the same indentation level as `def` 

(also the explicit `return` is not necessary)